projects/aca-testing-shared/src/components/info-drawer/info-drawer-comments-tab.ts

```diff
@@ -23,22 +23,23 @@
  * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { by, browser, until } from 'protractor';
+import { by, browser, until, ElementFinder } from 'protractor';
 import { Component } from '../component';
 import { BROWSER_WAIT_TIMEOUT } from '../../configs';
 import { typeText } from '../../utilities/utils';
 import { BrowserActions, BrowserVisibility } from '@alfresco/adf-testing';
+import { By } from '@angular/platform-browser';
 
 export class CommentsTab extends Component {
-  commentsContainer = this.byCss('.adf-comments-container');
-  commentsHeader = this.byCss('.adf-comments-header');
-  commentTextarea = this.byCss('.adf-comments-input-container textarea');
-  addCommentButton = this.byCss('button.adf-comments-input-add');
-  commentListItem = by.css('.adf-comment-list-item');
-  commentUserAvatar = by.css('.adf-comment-img-container');
-  commentUser = by.css('.adf-comment-user-name');
-  commentText = by.css('.adf-comment-message');
-  commentTime = by.css('.adf-comment-message-time');
+  commentsContainer: ElementFinder = this.byCss('.adf-comments-container');
```

Comment:
why do you need explicit typings here? it should be inferred by typescript already

Reply:
as discussed, TS had an issue with automatically infer the type, nevertheless as discussed, this lib wont be build